ProteGO/UI/Dashboard/View/DangerStatusCards/Template/DangerStatusCardBottomModule.swift

```diff
@@ -119,9 +121,11 @@ class DangerStatusCardBottomModule: UIView {
         self.descriptionSecondParagraphLabel.isUserInteractionEnabled = true
     }
 
-    private func setupContactButton(color: UIColor, text: String) {
+    private func setupContactButton(normalColor: UIColor, highlightedColor: UIColor, text: String) {
         self.contactButton.setBackgroundColor(.white, forState: .normal)
-        self.contactButton.setTitleColor(color, for: .normal)
+        self.contactButton.setBackgroundColor(highlightedColor, forState: .highlighted)
+        self.contactButton.setTitleColor(normalColor, for: .normal)
+        self.contactButton.setTitleColor(.white, for: .highlighted)
```

Comment:
Proponuję skorzystać z rozszerzenia `UIButton.rectButton`. Uważam, że zwiększyłoby to czytelność.